.gitignore

```diff
@@ -7,3 +7,5 @@ Gemfile.lock
 apps/**/translations/en
 coverage
 artefacts
```

Comment:
Is there a folder called artefacts?

Reply:
there's a whole bunch of stuff, I'll remove them